test/sidechain/L2Coordinator.spec.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
We can probably add a few more tests to this file, call queueNewRewards, mint etc.

Reply:
Done